Documentation/concepts/kubernetes/configuration.rst

```diff
@@ -47,6 +47,18 @@ to your preferences:
   packets that involve an L4 connection state change. Valid options are
   ``none``, ``low``, ``medium``, ``maximum``.
 
+  - ``none`` - Generate a tracing event on every receive and send packet.
+  - ``low`` - Generate a tracing event on every send packet.
+  - ``medium`` - Generate a tracing event on every new connection and connection
+    state change, and periodically generate tracing events on new sets of TCP
+    flags seen for existing TCP connections. The reporting interval for new TCP
+    flags is defined by ``monitor-aggregation-interval``.
+  - ``maximum`` - An alias for the most aggressive aggregation level. Currently
+    this is equivalent to setting ``monitor-aggregation`` to ``medium``.
+
+* ``monitor-aggregation-interval`` - Defines the interval to report tracing
+  events for new TCP flags with ``medium`` monitor aggregation level.
```

Comment:
this is a little bit confusing to me, setting interval also defines the level as medium ? or internal is only applicable for medium and above level ?

Reply:
Apologies, I often forget that this logic is actually pretty nuanced.

```suggestion
  events. Only applicable for ``monitor-aggregation`` levels ``medium`` or higher. Assuming new packets are sent at least once per interval, this ensures that on average one event is sent during the interval.
```

This is separate from the flags logic, the flags logic always generates a new event for each newly-seen flag for the direction. We can perhaps clarify this further as well.